framework/include/materials/DerivativeMaterialInterface.h

```diff
@@ -77,6 +77,20 @@ class DerivativeMaterialInterface : public T, public DerivativeMaterialPropertyN
                                                             const VariableName & c1,
                                                             const VariableName & c2 = "",
                                                             const VariableName & c3 = "");
+
+  template <typename U>
+  const MaterialProperty<U> &
+  getMaterialPropertyDerivative(const std::string & base,
```

Comment:
Probably a good idea to explain what the `v` parameters are up in the blanket doxygen comment for these methods up above.